src/mmw/js/src/modeling/views.js

```diff
@@ -326,7 +326,14 @@ var ScenarioTabPanelView = Marionette.ItemView.extend({
     },
 
     renameScenario: function() {
-        var self = this;
+        var self = this,
+            setScenarioName = function(name) {
```

Comment:
It doesn't _need_ to be, but since we're packaging local functionality for use twice locally (in response to https://github.com/WikiWatershed/model-my-watershed/pull/619#discussion_r35756091), I kept it local. I suppose it could be useful to have this outside, but I didn't have an immediate use case for that and wanted to contain my change to have a small footprint (keep it within `renameScenario`). If this is not satisfactory, then I can move it out.